plugins/labhub.py

```diff
@@ -302,6 +310,25 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
+            @register_check
+            def newcomer_issue_check(user, iss):
+                """
+                True if:  Issue is not labeled `difficulty/newcomer` and
+                          user is not a newcomer.
+                False if: A `difficulty/newcomer` issue is already assigned
+                          to the user.
+                """
+                if (self.is_newcomer_issue(iss)
+                    and self.TEAMS[self.GH_ORG_NAME +
+                                   ' newcomers'].is_member(user)):
+                    search_query = 'user:coala assignee:{} ' \
+                                   'label:difficulty/newcomer'.format(user)
+                    result = GitHub.raw_search(GitHubToken(
+                        os.environ.get('GH_TOKEN')), search_query)
+                    return not (sum(1 for _ in result) >= 1)
```

Comment:
`return len(list(result)) == 0`

Reply:
since IGitt returns a generator object, this is a better way to calculate length. I read it on StackOverflow https://stackoverflow.com/questions/7460836/how-to-lengenerator#comment29087013_7460986